### ISSUE TITLE
[filterscripts] MESE Filter Throws Key Error (Trac #2331)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2323]
This is with the latest trunk of combo r174135.

ERROR (I3Module): MeseFilter_precut: Exception thrown (I3Module.cxx:123 in void I3Module::Do(void (I3Module::*)()))
Traceback (most recent call last):
  File "/home/olivas/icecube/combo/trunk/build/filterscripts/resources/scripts/SimulationFiltering.py", line 387, in <module>
    main(opts)
  File "/home/olivas/icecube/combo/trunk/build/filterscripts/resources/scripts/SimulationFiltering.py", line 344, in main
    tray.Execute()
  File "/home/olivas/icecube/combo/trunk/build/lib/I3Tray.py", line 256, in Execute
    super(I3Tray, self).Execute()
  File "/home/olivas/icecube/combo/trunk/build/lib/icecube/filterscripts/mesefilter.py", line 127, in precut
    layer_veto_charge = frame['L4VetoLayer0'].value + frame['L4VetoLayer1'].value
KeyError: 'L4VetoLayer0'

Migrated from https://code.icecube.wisc.edu/ticket/2331
```json
{
    "status": "closed", 
    "changetime": "2019-06-28T22:27:43", 
    "description": "This is with the latest trunk of combo r174135.\n\nERROR (I3Module): MeseFilter_precut: Exception thrown (I3Module.cxx:123 in void I3Module::Do(void (I3Module::*)()))\nTraceback (most recent call last):\n  File \"/home/olivas/icecube/combo/trunk/build/filterscripts/resources/scripts/SimulationFiltering.py\", line 387, in <module>\n    main(opts)\n  File \"/home/olivas/icecube/combo/trunk/build/filterscripts/resources/scripts/SimulationFiltering.py\", line 344, in main\n    tray.Execute()\n  File \"/home/olivas/icecube/combo/trunk/build/lib/I3Tray.py\", line 256, in Execute\n    super(I3Tray, self).Execute()\n  File \"/home/olivas/icecube/combo/trunk/build/lib/icecube/filterscripts/mesefilter.py\", line 127, in precut\n    layer_veto_charge = frame['L4VetoLayer0'].value + frame['L4VetoLayer1'].value\nKeyError: 'L4VetoLayer0'\n", 
    "reporter": "olivas", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1561760863704892", 
    "component": "jeb + pnf", 
    "summary": "[filterscripts] MESE Filter Throws Key Error", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2019-06-22T02:23:41", 
    "milestone": "Autumnal Equinox 2019", 
    "owner": "blaufuss", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20190624T15:38:19: olivas commented:

> To the owner of this ticket.  Signal you're aware if this ticket, by 'accept'ing it.

**Comment by icecube-trac** [comment]
Trac update at 20190624T15:39:33: blaufuss changed status from "new" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20190628T22:27:43: 
* blaufuss commented:

> In changeset 174304:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="174304"
> restore the changes made to filterscripts trunk for 2019 filter testing.  This should fix #2323
> ```


* blaufuss changed status from "accepted" to "closed"
* blaufuss changed resolution from "" to "fixed"
